.pre-commit-config.yaml

```diff
@@ -34,6 +34,11 @@ repos:
         language: system
         entry: >
           bash -c 'pnpm run format'
+      - id: cargo-format
+        name: cargor-format
```

Comment:
Is this a typo : `cargo-format` or `cargor-format`?

Reply:
Thanks. Resolved in 922c625ed2590bd81316123050ef8ba115427a29.